src/collection.ts

```diff
@@ -196,7 +196,35 @@ export class Collection<T extends TigrisCollectionType> {
 		});
 	}
 
-	read(
+	findMany(
+		filter: SelectorFilter<T> | LogicalFilter<T> | Selector<T>,
+		readFields ?: ReadFields,
+		tx ?: Session,
+		options ?: ReadRequestOptions
+	): Promise<Array<T>> {
+		return new Promise<Array<T>>((resolve, reject) => {
+			if (options === undefined) {
+				options = new ReadRequestOptions();
+				options.skip = 0;
+				options.limit = 100;
```

Comment:
can you define it somewhere?

Reply:
I am thinking to document it in client reference doc.

Reply:
better to define "defaultLimit" somewhere, btw what will happen if only skip is passed in the options or options are created with empty values(both set as 0)

Reply:
updated the defaults and overloaded constructor.